tests/Types/JsonTest.php

```diff
@@ -62,7 +62,7 @@ public function testJsonEmptyStringConvertsToPHPValue(): void
     public function testJsonStringConvertsToPHPValue(): void
     {
         $value         = ['foo' => 'bar', 'bar' => 'foo'];
-        $databaseValue = json_encode($value, 0, JSON_THROW_ON_ERROR);
```

Comment:
This should probably stay as it was, since `JSON_THROW_ON_ERROR` has an effect on how `vimeo/psalm` analyzes this code

Reply:
Reverted back